scoda/static/src/sass/components/_about-us.scss

```diff
@@ -16,9 +16,7 @@
         line-height: 28px;
         font-weight: 400;
     }
-    img{
-        width: 100%;
-    }
+
```

Comment:
@PhakamaniXulu the spacing and indentation in this file is not good at all. Please leave code better than you found it.